### PR TITLE
changes

### PR DESCRIPTION
**Pull request by 12944qwerty**
Before merging, make sure to update the replugged dep compat version.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,25 +6,22 @@
...
-    "@parcel/core": "^2.8.2",
-    "@parcel/transformer-sass": "^2.8.2",
+    "@parcel/config-default": "^2.8.3",
+    "@parcel/core": "^2.8.3",
     "@types/node": "^18.11.15",
-    "parcel": "^2.8.2",
     "prettier": "^2.8.1",
-    "replugged": "4.0.0-beta0.17",
+    "replugged": "^4.0.0-beta0.22",
     "tsx": "^3.12.1"
```
Comment (asportnoy) [inline package.json]: tsx and parcel are not needed anymore

File: package.json [hunk 2]
```diff
@@ -6,25 +6,22 @@
...
-    "build": "tsx scripts/build.ts",
-    "watch": "tsx scripts/build.ts --watch",
-    "bundle": "tsx scripts/bundle.ts",
-    "build-and-bundle": "pnpm run build --no-install && pnpm run bundle",
-    "lint": "prettier ./src ./scripts --check",
-    "lint:fix": "prettier ./src ./scripts --write"
+    "build": "replugged build theme",
+    "watch": "replugged build theme --watch",
+    "bundle": "replugged bundle",
+    "build-and-bundle": "replugged build theme --no-install && replugged bundle",
```
Comment (asportnoy) [inline package.json]: Per comment in main PR
```suggestion
```

**Review by asportnoy — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (12944qwerty) [reply package.json]: for some reason, it requires parcel to be installed. It won't work without it.

**Review by asportnoy — APPROVED** [review]
image

**Pull request merged**